raincloud/__init__.py

```diff
@@ -76,18 +77,21 @@ def restoreBackup():
         # call the ~/project_rainstorm/scripts/backups_restore.sh script
         return 
 
-    # TODO: get list of services
-    @app.route('/service/list', methods=['GET'])
+    @app.route('/services', methods=['GET'])
```

Comment:
If we used Flask-JSON, we could replace these json.dumps by using the @as_json decorator.